canonical-websites/www.canonical.com
# Point careers feed through proxy

## Done

* Add proxy to careers feed if debug mode is not on
* Revert ./run script which is required to deploy site with requirements.txt

## QA

1. Check out this feature branch
2. Run the site using the command `./run`
3. View the site locally in your web browser at: http://0.0.0.0:8002/careers/all-vacancies
4. See the content load correctly
5. Remove cache.sqlite from root folder and remove DJANGO_DEBUG from .env
6. Start server back up and the same URL will error. (If you have a proxy lying around,  you could test a little with that.)
